src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
However, there is a fundamental problem with this approach -- namely that JsonObject doesn't let you add a property of type Date.  Thus my Vert.x newbie-ness comes to the fore -- I guess I was expecting a Jackon like binding here, either to Date or Joda or... Am I missing something?